aiohttp/multipart.py

```diff
@@ -907,16 +901,14 @@ def write(self, chunk):
                     return
 
         if self._encoding == 'base64':
-            self._encoding_buffer.extend(chunk)
-
-            if self._encoding_buffer:
-                buffer = self._encoding_buffer
-                div, mod = divmod(len(buffer), 3)
-                enc_chunk, self._encoding_buffer = (
-                    buffer[:div * 3], buffer[div * 3:])
-                if enc_chunk:
-                    enc_chunk = base64.b64encode(enc_chunk)
-                    yield from self._writer.write(enc_chunk)
+            if chunk:
+                enc_chunk = base64.encodebytes(chunk)
```

Comment:
Are you serious?

Base64-encoding adds equal-signs at the end for alignment. So two single-byte chunks like `A` will write `QQ==QQ=` instead of `QUE=`.